index.html

```diff
@@ -138,6 +141,13 @@ <h2>Pairings</h2>
         </div>
       </div>
 
+      <div id="myModal" class="modal">
```

Comment:
Your using hyphens for other class names and camel casing for this id I'd reccomend sticking with one or the other

Reply:
will do!